routes/libraries.js

```diff
@@ -175,7 +175,8 @@ router.route('/:id')
     }
 
     try {
-      res.render('libraries/library', { title: library.name, library: library, isLoggedIn: true, script_partial: 'comment', userid: req.session.user._id});
+      let user = req.session.user;
+      res.render('libraries/library', { title: library.name, isLoggedIn: true, script_partial: 'comment', userid: user._id, ...library});
```

Comment:
user should be saved as a global variable so you can access directly through handlebars without passing it thru

Reply:
What do you mean?